Version 2.2 Fork / Login Form

mkkeck opened the pull request:
* Login form
* Added session class for login form
* Display different navigations if user is logged in / logged out

mkkeck committed:
Version 2.2 Fork / Login Form

* Login form
* Added session class for login form
* Display different navigations if user is logged in / logged out

mkkeck committed:
Version 2.2 Fork / Description for login details in config

Better description for login details in config_example.php

mkkeck committed:
Version 2.2 Fork / Update README.md

* Readme section restructured
* Config / Install description updated for admin login
* Linking to Issues
* Linking to original project from Christian Beer

mkkeck committed:
Version 2.2 Fork / Update README.md

* Readme section restructured
* Config / Install description updated for admin login
* Linking to Issues
* Linking to original project from Christian Beer

mkkeck committed:
Version 2.2 Fork / Update README.md

* Readme section restructured
* Config / Install description updated for admin login
* Linking to Issues
* Linking to original project from Christian Beer

ChristianBeer commented:
Hi, do you want to create your own fork or is this meant to be merged and released as version 2.3? I just glanced at the changes which look good in general. I will have more time to review this later this week. I want to take a closer look at the session handling.

mkkeck committed:
Version 2.2 Fork / Secure config and includes directories

* Added security for Apache Webserver:
  - `configs/`  not accessible via browser
  - `includes/` not accessible via browser
* Moved `tables*.sql` to `install/tables*.sql`
* Added `install/config-examples` for Linux Distributions
* Added `install/config-examples/debian` as example and info how to setup on Debian Jessie

mkkeck commented:
I've created my own fork. But it would be also ok to be merged and released as new version.
The session handling may be secure enough, it is based on the ideas from other great admin tools like phpMyAdmin, Roundcube etc. But such admin tools should be accessed only via https. Perhabs it would be needed to document and forced via the tool itself.

mkkeck committed:
Version 2.2 Fork / Security: option to force SSL

* Config parameter $cfg['force_ssl']
* If $cfg['force_ssl'] true, then redirect to https://
* Some typos fixed

mkkeck committed:
Version 2.2 Fork / Example Configs for CentOS 7.3

mkkeck committed:
Version 2.2 Fork / Example Configs for CentOS 7.3

mkkeck committed:
Update config-example.php

mkkeck committed:
Version 2.2 Fork / Example Configs for CentOS 7.3: DB Settings

mkkeck committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	install/config-examples/centos-7.3/config-example.php

Commit:
Change Password defaults to crypt + SAH512, Add User Expiry Date

Commit:
bootstrap-datetimepicker

Commit:
Added SSH public key fields and columns

Commit:
Added sftp configs

Commit:
Add SSH Key support

Commit:
update to CRLF

update to CRLF

Commit:
Change Style

Commit:
CSS und variable anpassung

Commit:
Anpassung für php8.x

Commit:
php8.1 and typing errors

Commit:
php8.1 and typing errors

Commit:
Add fix expiration

mkkeck committed:
Merge pull request #3 from preussal/master

Merged php8.1 and crypt

jniltinho committed:
Update tables.sql

Error import Mysql line 40

mkkeck committed:
Merge pull request #4 from jniltinho/patch-1

Update tables.sql

mkkeck committed:
Add test.md with various Markdown examples
